extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -50,26 +55,51 @@ export class AmpStoryRequestService {
    * @private
    */
   loadBookendConfigImpl_() {
-    return this.loadJsonFromAttribute_(BOOKEND_CONFIG_ATTRIBUTE_NAME);
+    const bookendEl = childElementByTag(this.storyElement_,
+        'amp-story-bookend');
+    if (!bookendEl) {
+      return Promise.resolve(null);
+    }
+
+    if (bookendEl.hasAttribute(BOOKEND_CONFIG_ATTRIBUTE_NAME)) {
+      return this.loadJsonFromAttribute_(
+          BOOKEND_CONFIG_ATTRIBUTE_NAME, bookendEl);
+    }
+
+    return this.loadInlineJson_(bookendEl);
```

Comment:
Nit: because it's hard to justify why a `loadInlineJson` method lives in the `request-service`, I think we should keep it the code in this very method, like:

````
let config = null;
try {
  config = getChildJsonConfig(bookendEl);
} catch (err) {}

return Promise.resolve(config);
````